_author_instructions/01_policies.md

```diff
@@ -205,16 +201,17 @@ After this time, a revised manuscript may be handled as a new submission.
 
 ## Upon acceptance
-After acceptance of your article, the first task is to prepare an appropriate "ASAP" version of the article which will be posted to the journal's website prior to publication of the next issue. Several steps, outlined below, are required. Note that ASAP version will essentially be final, as only minor corrections as detailed below will be permitted prior to publication as part of an issue.
 
-- Make sure the metadata is complete in the submission (including institution, ORCID ID, abstract). 
-- Have the managing editor assign a DOI to the article (correct metadata is needed for this).
-- Do a final check for issues with grammar, word choice, spelling, formatting (including issues with the LaTeX template, placement of figures, etc.) and typos, as normally would be done at the proofs stage. Please [reference the appropriate part of the author instructions](https://livecomsjournal.github.io/authors/policies/#writing-style-and-editing) for what to look for. Note that authors bear ultimate responsibility for ensuring the writing is of good quality. Editors may suggest improvements, but this is at their discretion.
-- Enable the LiveCoMS footer by including the `pubversion` class option in the document preamble, fill in the `\datereceived`, `\dateaccepted`, and `\pubDOI` fields, which the managing editor will provide after the Metadata is verified to be correct. You should make the PDF accessible on the GitHub repository as `releases/LiveCoMS_Article_VX.pdf`, where 'X' is the current article version.  The related header image should be at least 1200 pixels wide and in `.jpg` format, and should be available on the GitHub repository as `releases/header_VX.jpg`, where again 'X' is the current article version, and notify the editor when this has been done. 
+After acceptance of your article, the first task is to prepare an appropriate "ASAP" version of the article which will be posted to the journal's website prior to publication of the next issue. Several steps, outlined below, are required. Note that ASAP version will essentially be final, as only minor corrections as detailed below will be permitted prior to publication as part of an issue.
+- Verify that the metadata is complete in the submission. This includes institutions with departments (if applicable) for all authors, ORCID ID for all authors, and a correctly formatted abstract.
```

Comment:
So I am unclear where we would verify the metadata is complete "in the submission", and I just went through sending my paper through. Specifically I just got an e-mail saying my paper was accepted, and then I'm supposed to generate a PDF and send it to you. Are you saying I should log back in to Scholastica and check that the metadata is correct in the Scholastica site? If so perhaps make more clear. 

Worse than that, I just logged in to Scholastica, went in to my paper, and I can see that it lists all the authors, and MY ORCID, but I cannot see any way to add the other authors' ORCIDs (or, for that matter, edit anything -- which would sort of make sense since my paper was accepted). It also doesn't list departments.

So, what am I supposed to do? E-mail all of that to you or to the managing editors? 

(I did also visit someone else's manuscript as an editor rather than an author, and I also do not see how to update this info there.)

Reply:
> Worse than that, I just logged in to Scholastica, went in to my paper, and I can see that it lists all the authors, and MY ORCID, but I cannot see any way to add the other authors' ORCIDs (or, for that matter, edit anything -- which would sort of make sense since my paper was accepted). It also doesn't list departments.

Yeah, this information can't be fixed by an author post-submission. That has to be fixed by an editor. Do you see your department?  I see it when I look as an editor. I think you were the only author that had one entered when you submitted.   The pyEMMA manuscript has departments for everyone when I look at it as an editor.

> (I did also visit someone else's manuscript as an editor rather than an author, and I also do not see how to update this info there.)

It cannot be edited in the reviewing half of Scholastica - you have to publish a new article, import the accepted article, and then enter the information there.  I should do a walkthrough with people the next article to come through.  Look at the instructions here for now: http://help.scholasticahq.com/customer/en/portal/articles/1049531-how-does-publishing-work-on-scholastica-?t=565043

Reply:
I can't see it even if i go in as editor, but maybe that's because I'm not the editor assigned to any of the papers.

So... your author instructions here need clarification since it does not currently make clear what they should do if the metadata is not complete. Sounds like the answer is "e-mail the editor".

Reply:
No, being editor of a paper is irrelevant.  You have to view it as a publisher, going into the "publish article" part of the site, which is a completely different tab.
<img width="845" alt="bar_scholastica" src="https://user-images.githubusercontent.com/4274132/49174164-d4c58a00-f302-11e8-8a62-de36b1f7e149.png">

Reply:
Yes.  

Though perhaps real thing to do is reject and revise the paper if they never put in the metadata to being with.

Reply:
OK. But I'm pretty sure the submission site only allowed for the ORCID of the lead author. Not 100% sure though.

Reply:
All fields are the same for all authors (I just opened up a manuscript and checked).